packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -428,6 +430,34 @@ export const HelpCenterContactForm = () => {
 		return isSubmitting ? formTitles.buttonSubmittingLabel : formTitles.buttonLabel;
 	};
 
+	// TODO: Figure out in which environments this makes sense.
+	const showGPTResponse = true;
```

Comment:
The PR mentions adding `show-results=true` to the URL - I assume you're planning to use a feature flag with `isEnabled`?